controllers/component_build_controller.go

```diff
@@ -342,6 +407,11 @@ func (r *ComponentBuildReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 			}
 		} else {
 			simpleBuildStatus.BuildStartTime = time.Now().Format(time.RFC1123)
+			simpleBuildPipelineCreationTimeMetric.Observe(time.Since(componentTimesForMetrics[componentIdForMetrics].startTimestamp).Seconds())
+			// initial build upon component creation (doesn't have either build status)
+			if initialBuildStatus.PaC == nil && initialBuildStatus.Simple == nil {
```

Comment:
Suggest not use words "initial build" here to avoid possible confusion, as build-service has a concept of initial build. `buildStatus` should be enough to describe what the variable references.

Reply:
I'd say 'initial build upon component creation' isn't confusing, also I wanted to make it clear for everybody

Adam wanted more clear explanation and you want less

what I have is good I'd say